runelite-api/src/main/java/net/runelite/api/Client.java

```diff
@@ -1575,4 +1575,10 @@
 	int getRasterizer3D_clipMidY2();
 
 	void checkClickbox(Model model, int orientation, int pitchSin, int pitchCos, int yawSin, int yawCos, int x, int y, int z, long hash);
+
+	/**
+	 * Returns the Afflictions currently affecting the local player
+	 * @return
```

Comment:
Better to just leave this out if you're not adding a description of it. Realistically though, the whole javadoc for this entry could be:

```
/**
 * Gets afflictions affecting the local player
 * @return A set of {@link Affliction}s affecting the local player.
 */
```